packages/backend/src/clients/EmailClient/EmailClient.ts

```diff
@@ -165,7 +165,8 @@ export default class EmailClient {
                 roleAction = 'manage';
                 break;
             default:
-                const nope: never = projectMember.role;
+                // NOOP - Intentionally left blank
```

Comment:
Hey @filp, This call checks whether the switch/case is "exhaustive" or not. Was this change esaential?

Fyi, We can use the `assertUnreachable` utility function from common.

Reply:
Was this supposed to raise an `eslint` error? I agree with using `assertUnreachable`, but didn't know it was available under `common` and didn't want to introduce new exceptions as part of this changeset.

Agreed overall though, I'll make the adjustments!

Reply:
good point! if we don't want to raise then maybe we could do `_nope` or `nope_`?
but then again, looks like a code block that should never ever happen and its better to throw in that case